installer/datafiles/base_omsagent.data

```diff
@@ -67,6 +67,11 @@ MAINTAINER:              'Microsoft Corporation'
 /opt/microsoft/omsagent/plugin/out_oms_api.rb;                          source/code/plugins/out_oms_api.rb;                    744; root; root
 /opt/microsoft/omsagent/plugin/in_sudo_tail.rb;                         source/code/plugins/in_sudo_tail.rb;                   744; root; root
 
+/opt/microsoft/omsagent/plugin/filter_security_baseline.rb;             source/code/plugins/filter_security_baseline.rb;       744; root; root
+/opt/microsoft/omsagent/plugin/security_baseline_lib.rb;                source/code/plugins/security_baseline_lib.rb;          744; root; root
+/opt/microsoft/omsagent/plugin/omsbaseline;                             source/code/plugins/omsbaseline;                       755; root; root
```

Comment:
Change "source/code/plugins/omsbaseline" to "source/code/plugins/omsbaseline_${{PFARCH}}".
No need to alter the Makefile.